src/core/stream_send.c

```diff
@@ -400,6 +400,8 @@ QuicStreamCompleteSendRequest(
             SendRequest->InternalBuffer.Length);
     }
 
+    // FIXME: this underflows when running spinquic
+    QUIC_DBG_ASSERT(Stream->Connection->SendBuffer.PostedBytes >= SendRequest->TotalLength);
```

Comment:
This is interesting. Let me look into this some... Can you open an issue for this?

Reply:
Done: #719